hazelcast/src/main/java/com/hazelcast/sql/impl/SqlDataSerializerHook.java

```diff
@@ -155,8 +156,9 @@
     public static final int EXPRESSION_CONCAT = 58;
     public static final int EXPRESSION_LIKE = 59;
     public static final int EXPRESSION_SUBSTRING = 60;
+    public static final int EXPRESSION_TRIM = 61;
 
-    public static final int LEN = EXPRESSION_SUBSTRING + 1;
+    public static final int LEN = EXPRESSION_TRIM + 1;
```

Comment:
Wouldn't it be easier to use ArrayList to build the constructor list and then use toArray on it and avoid this `LEN` constant? Seems like micro-optimization.

Reply:
Perhaps you are right, it would be easier. But this is a common approach across other hooks, we just follow it here.